src/Components.tsx

```diff
@@ -32,18 +32,29 @@ export const ConnectPaymentDetails = ({
 };
 
 export const ConnectAccountOnboarding = ({
-  onOnboardingExited,
+  onExit,
+  recipientTermsOfServiceUrl,
+  fullTermsOfServiceUrl,
 }: {
-  onOnboardingExited: () => void;
+  onExit: () => void;
```

Comment:
Let's make sure we don't deploy the `onExit` change before that change to `submerchant-surfaces` is deployed.

Reply:
Hey @jorgea-stripe / @jojo-stripe 

We currently listen for `onOnboardingExited`. When the service is deployed to change to `onExit`, will our code break even if we don't upgrade our `react-connect-js` version?

Reply:
No, we will be emitting both events for backwards compatibility!

Reply:
Beautiful! 😄 Thanks for the reply